### PR TITLE
fix: onboard-pricing-options-self-host

### LINKED ISSUE #10075: [CAL-2149] Remove EE plan for self host onboarding
Linear-hosted image

[Source image](https://uploads.linear.app/e86bf957-d82f-465e-b205-135559f4b623/bcfff58b-47bb-4294-8f98-d8a596062c68/85a655bf-c34d-42ed-86b9-9a98dbc509b8)

Should redirect to cal.com/sales and not be an option for setup wizard

<sub>From [SyncLinear.com](https://synclinear.com) | [CAL-2149](https://linear.app/calcom/issue/CAL-2149/remove-ee-plan-for-self-host-onboarding)</sub>

### PR DESCRIPTION
**Pull request by alannnc**
## What does this PR do?

- Change $99 plan with a link to contact sales page.

Fixes #10075 
## Requirement/Documentation

image

## Type of change

  - [X] Chore (refactoring code, technical debt, workflow improvements)

## How should this be tested?

  - Are there environment variables that should be set? No.
  - What are the minimal test data to have? DB without any user admin created.
  - What is expected (happy path) to have (input and output)? 
  Button should redirect to cal.com/sales pages instead of being selectable.
  - Any other important info that could help to test that PR
  Modifying/Commenting code on the  SSR function that returns 404 when an user admin exist on the DB on auth/index so auth/setup doesn't return 404

## Mandatory Tasks

  - [X] Make sure you have self-reviewed the code. A decent size PR without self-review might be rejected.

## Checklist

- I haven't added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: apps/web/components/setup/ChooseLicense.tsx [hunk 1]
```diff
@@ -1,11 +1,10 @@
 import * as RadioGroup from "@radix-ui/react-radio-group";
 import classNames from "classnames";
+import Link from "next/link";
 import { useState } from "react";
 
 import { useLocale } from "@calcom/lib/hooks/useLocale";
 
-const ENTERPRISE_BOOKING_FEE = "$99"; // TODO: get this from a new API endpoint
```
Comment (alannnc) [inline apps/web/components/setup/ChooseLicense.tsx:7]: No longer needed

File: apps/web/components/setup/ChooseLicense.tsx [hunk 2]
```diff
@@ -50,23 +49,18 @@ const ChooseLicense = (
             </ul>
           </div>
         </RadioGroup.Item>
-        <RadioGroup.Item value="EE">
```
Comment (alannnc) [inline apps/web/components/setup/ChooseLicense.tsx:53]: Disabled and now as a Link

**Review by zomars — APPROVED** [review]
Ship it

**Comment by alannnc** [comment]
> Is there a screenshot to see the new panel?

On the PR description

**Pull request merged**